pandas/_libs/tslibs/nattype.pyx

```diff
@@ -1279,6 +1281,11 @@ class NaTType(_NaT):
         >>> pd.Timestamp.now()  # doctest: +SKIP
         Timestamp('2020-11-16 22:06:16.378782')
 
+        If you want a specific timezone, in this case 'Brazil/East':
+
+        >>> pd.Timestamp.now('Brazil/East')
+        Timestamp('2025-11-11 22:17:59.609943-03:00)
```

Comment:
This is going to be problematic with our doctests, as it will compare the result from the code with what is hardcoded here. Unfortunately these aren't reliably running currently. I suppose you could add `# doctest: +SKIP` to the first line here.